### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by michaelcychan**
1. Readme updated.
2. Removed obsolete or commented codes
3. Tidied up formatting

All unit tests and integration tests passed.

### CODE REVIEW

File: controllers/users.js [hunk 1]
```diff
@@ -10,22 +10,6 @@ const UsersController = {
...
-  //         error: 'User already exists!', 
-  //         name: req.name, 
-  //         surname: req.surname});
-  //     } else {
-  //     res.status(201).redirect("/posts");
-  //     }
-  //   });
-  // },
-
   Create: (req, res) => {
```
Comment (kaykakaraka) [inline controllers/users.js:13]: nice clearing up of files

File: README.md [hunk 2]
```diff
@@ -24,6 +21,23 @@ https://github.com/fridayshoes/acebook-node-priceless
...
+   - A post can have multiple lines, it is displayed correctly on wall
+- User can delete a post
+- Users can react to a post by 
+   - liking a post (each user can only like a post once)
+   - commenting on a post
+- User has a Profile Page
+   - User can make posts to another user on Profile Page
+   - User can see posts made by themselves and made to them on Profile Page
+- User passwords are encrypted before they are stored in the database
+
```
Comment (kaykakaraka) [inline README.md:39]: readme looks good

**Review by kaykakaraka — APPROVED** [review]
Readme looks good and code looks a bit cleaner! Do we no longer need the loginCheck?

**Review by MalikOJBS — APPROVED** [review]
good job cleaning the code

File: README.md (hunk 2 above)
Comment (fridayshoes) [reply README.md:39]: - Dates are added to posts
- Posts are sorted with newest post first

**Review by fridayshoes — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (michaelcychan) [reply README.md:41]: @fridayshoes Added both.

**Review by fridayshoes — APPROVED** [review]

**Pull request merged**